EIPS/eip-7748.md

```diff
@@ -64,12 +64,12 @@ class StoragePhase:
     The account next conversion step continues converting the 
     storage-slot with key greater or equal next_key. 
     If there isn't such storage-slot, the account must move to
-    BasicDataPhase.
+    AccountDataPhase.
     """
     next_key: Bytes   
    
 @dataclass
-class BasicDataPhase:
+class AccountDataPhase:
```

Comment:
With @gabrocheleau, we decided to rename it `AccountDataPhase` since `BasicDataPhase` is a bit off, considering that in VKT, it usually leaves out the code-hash (which, in case of the migration, is also moved).

We'd like at least one extra approval for this renaming (or new suggestion!) from another author.

cc @gballet @g11tech @tanishqjasoria

(Note: avoiding switching from Draft to avoid automerge)